README.md

```diff
@@ -68,14 +68,12 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 |---|---|
 |**Read** access to code & metadata | To sync development information to Jira|
 |**Read** and **write** access to issues and pull requests| To power Smart Commit actions and unfurl Jira URLs|
+|**Read** and **write** access to actions and deployments| To sync CI/CD (GitHub Actions) information to Jira|
```

Comment:
We need to improve this explanation on permissions - but we have a different ticket to do so [ARC-677]